api/game.ts

```diff
@@ -1,27 +1,26 @@
 import { httpClient } from "./httpClient";
-import { TOKEN } from "../helpers/constants";
 import { Game } from "./interface";
 
 export const createGame = async (): Promise<Game> => {
-  const result = await httpClient.post<Game>("/game", {}, TOKEN);
+  const result = await httpClient.post<Game>("/game", {});
```

Comment:
Dont need to send empty params either. Had added that because we were passing token